src/Services/Air/AirParser.js

```diff
@@ -540,33 +540,41 @@ function extractBookings(obj) {
       })
       : [];
 
+    const fareQuotesCommon = {};
```

Comment:
It's common data not for fare quotes, but for pricingInfos, which will be distributed between fareQuotes